Counties/Palm Beach County/Makefile

```diff
@@ -0,0 +1,8 @@
+init:
```

Comment:
why is this not in a state folder?

Reply:
Probably should be, but this was the first scraper checked in (hence why it's in "Counties"). I don't think "Counties" is quite right, either, though, since some cities will likely have their own distinct records. Maybe call them "municipalities" or something? IDK, but that's probably outside the scope of this PR. Filed #97